warehouse/query-core/src/test/java/datawave/query/TestLimitReturnedGroupsToHitTermGroups.java

```diff
@@ -276,6 +300,7 @@ public void testOneGroup() throws Exception {
         Set<String> goodResults = Sets.newHashSet("CANINE.PET.13:shepherd", "CAT.PET.13:ragdoll", "FISH.PET.13:tetra", "BIRD.PET.13:lovebird",
                         "REPTILE.PET.1:snake", "DOG.WILD.1:coyote", "SIZE.CANINE.3:20,12.5", "SIZE.CANINE.WILD.1:90,26.5");
 
+        // TODO: when executing as a document range no results are found
```

Comment:
meant to delete this line since it works now

Reply:
done